openapi/components/schemas/Website.yaml

```
@@ -85,6 +85,10 @@ properties:
                 nullable: true
                 items:
                   type: string
+              fullPageRedirect:
+                type: boolean
+                description: Specifies if the hosted payment form should be a whole page redirect for approval url redirects, or use the default iframe modal
```

suggestion: to avoid passive (should)
```suggestion
                description: Specifies whether the hosted payment form uses a whole page redirect or the default iframe modal for approval URL redirects.
```

---

Prop name is `fullPageRedirect`, description is about whole page redirect. Is it expected?

---

Good point. I updated my suggestion also